proto/select.proto

```diff
@@ -80,6 +82,10 @@ message DAGRequest {
 
 	// It represents whether we collect the detailed scan counts in each range.
 	optional bool collect_range_counts = 6;
+
+	// It indicates the maximum number of error, warning, and note messages,
+	// and thus the number of messages that SHOW WARNINGS displays.
+	optional uint64 max_error_count = 7;
```

Comment:
It seems we only return one error now?

Reply:
@AndreMouche It does not only mean the count of error.
[This](https://dev.mysql.com/doc/refman/5.5/en/show-warnings.html) can be referred.